amplipi/updater/static/js/update-ui.js

```diff
@@ -71,9 +74,13 @@ function ui_reboot_app() {
       setTimeout(ui_check_after_reboot, 3000)
         // TODO: on fail -> show info on how to recover
     } else {
-      ui_add_log('Error restarting update server: ' + response, 'danger')
+      ui_add_log('Error restarting update server: ' + response, 'danger');
+      ui_show_failure();
```

Comment:
In this kind of place doing
```suggestion
      throw response;
```
makes more sense to me, but it doesn't really matter I suppose.

Reply:
Ok so the suggested changes feature gives less context of the surrounding code than I expected, but throwing the response there would get caught by another catch that would then log the same thing.